app/org/sagebionetworks/bridge/models/schedules/ScheduleContext.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.Map;
 import java.util.Objects;
+import java.util.Set;
```

Comment:
No tests for this class because this is being added in an other PR, and just stubbed out here so I can work on this independently.